src/Features/Blockcore.Features.Wallet/Database/WalletStore.cs

```diff
@@ -185,6 +203,7 @@ public IEnumerable<WalletHistoryData> GetAccountHistory(int accountIndex, bool e
             // The result of this method is not guaranteed to be the length
             //  of the 'take' param. In case some of the inputs we have are
             // in the same trx they will be grouped in to a single entry.
+            using var conn = this.GetDbConnection();
```

Comment:
this method is an example where you can reuse connection
e.g. if you write `conn.Open()` (not checked the proper syntax, I'm writing this comment without IDE) it will close the connection when you dispose (to be 100% safe, would be better in that case to explicitly call conn.Close() )